django/contrib/admindocs/views.py

```diff
@@ -206,7 +205,7 @@ def get_context_data(self, **kwargs):
         for field in opts.fields:
             # ForeignKey is a special case since the field will actually be a
             # descriptor that returns the other object
-            if isinstance(field, models.ForeignKey):
+            if field.many_to_one:
```

Comment:
I think `opts.fields` filters out GFK unless I read this incorrectly: https://github.com/django/django/blob/96ec67a7cf89a136e793305343c5bba8521cdb47/django/db/models/options.py#L380-L383